cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen
+
+import org.neo4j.cypher.docgen.tooling._
+
+class ShowProceduresTest extends DocumentingTest {
+
+  override def outputPath = "target/docs/dev/ql/listing"
+
+  override def doc: Document = new DocBuilder {
+    doc("Listing procedures", "query-listing-procedures")
+    p("Listing the available procedures can be done with `SHOW PROCEDURES`.")
+    p("include::listing-procedures-syntax.asciidoc[]")
+    p("This command will produce a table with the following columns:")
+    p("include::listing-procedures-columns.asciidoc[]")
+    section("Examples") {
+      section("Listing all procedures") {
+        p(
+          """To list all available procedures with the default output columns, the `SHOW PROCEDURES` command can be used.
+            |If all columns are wanted, use `SHOW PROCEDURES YIELD *`.""".stripMargin)
+        query("SHOW PROCEDURES", ResultAssertions(p => {
+          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
+        })) {
+          resultTable()
+        }
+      }
+      section("Listing procedures with filtering on output columns") {
+        p(
+          """The listed procedures can be filtered in multiple ways, one way is to use the `WHERE` clause.
+            |For example getting the name of all admin procedures:""".stripMargin)
+        query("SHOW PROCEDURES YIELD name, admin WHERE admin", ResultAssertions(p => {
+          p.columns should contain theSameElementsAs Array("name", "admin")
+          p.columnAs[Boolean]("admin").foreach(_ should be(true))
+        })) {
+          resultTable()
```

Comment:
Maybe swap the columns here would be nicer:
![image](https://user-images.githubusercontent.com/1756429/117288933-f1673400-ae63-11eb-9335-9ddb5c91a490.png)

Reply:
That is a problem all over and comes from me not wanting every example to have a return clause and therefore getting the columns in alphabetical order. This one I think could have it though